lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -135,6 +111,83 @@ class QueryBuilder
      */
     private $boundCounter = 0;
 
+    /**
+     * The SELECT parts of the query.
+     *
+     * @var string[]
+     */
+    public $select = [];
```

Comment:
In this case, it should be `private`. We made the other class'es properties public to avoid having to introduce getters but here it's not necessary.

Reply:
Oops, obviously. Copy/paste mistake. Fixed.